src/Jank/Scorepads/ScorepadList.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useReducer } from 'react';
+import useScorepads from '../../hooks/useScorepads';
+import ScorepadItem from './Scorepad';
+import { setScorepads } from './actions';
+import reducer from './reducer';
+import { ErrorText } from './ui';
+
+const ScorepadList = () => {
+  const [error, apiScorepads] = useScorepads('JanK');
+  const [scorepads, dispatch] = useReducer(reducer, []);
+
+  useEffect(() => {
+    if (apiScorepads !== null) {
+      dispatch(setScorepads(apiScorepads));
+    }
+  }, [apiScorepads]);
+
+  scorepads.reverse();
```

Comment:
Why do we need this? Whenever you are mutating an existing object/array that is coming from the outside (prop, state, hooks), you are probably breaking something ;) The reason being that mutations of existing objects are **not** detected as actual state changes. And React will only *react* to such changes. In this case, the reducer is not aware that the `scorepads` were changed, causing inconsistencies in the state tree compared to what is actually being displayed.

Another side effect: Since you are doing this on render, you are practically reversing the array whenever the component re-renders. I haven't tried it locally, but I am fairly certain that the order of scorepads will change all the time, whenever you make a change to the list of scorepads (e.g. 1-2-3-4 becomes 4-3-2 when deleting scorepad 1).

If (!) you need to reverse the array, you likely need to do it at an earlier step and outside the render function. You could do it, for example, in `useScorepads()` or in `setScorepads()` or something alike.
However, since you are probably not really interested in reversing the scorepads, I suggest to be a bit more explicit. I assume you want the scorepads to be sorted by date, so I'd suggest to do exactly that. You might want to do this in the `useScorepads()` hook already, so that we already have the scorepads in the right order when getting them from the hook.